pkg/skaffold/deploy/kpt/kpt.go

```diff
@@ -188,6 +200,27 @@ func (k *Deployer) renderManifests(ctx context.Context, _ io.Writer, builds []bu
 		return nil, fmt.Errorf("reading config manifests: %w", err)
 	}
 
+	// A workaround for issue https://github.com/GoogleContainerTools/kpt/issues/1149
```

Comment:
Thanks for adding this comment. 
I am a bit hesitant on adding a workaround for this bug, if #1149 is being actively worked on. 

I am assuming, this code will be removed once #1149 is fixed is that right?

Reply:
Skaffold and kpt have different release cycle. We can revert it once the kpt fix is in. But it won't be done easily (IMO). For this release, the workaround is necessary to avoid kpt deployer from broken.